css/src/atomics/typography.scss

```diff
@@ -0,0 +1,112 @@
+// Heading sizes
+.heading-h1 {
+	font-size: $font-size-1 !important;
+}
+
+.heading-h2 {
+	font-size: $font-size-2 !important;
+}
+
+.heading-h3 {
+	font-size: $font-size-3 !important;
+}
+
+.heading-h4 {
+	font-size: $font-size-4 !important;
+}
+
+.heading-h5 {
+	font-size: $font-size-5 !important;
+}
+
+.heading-h6 {
+	font-size: $font-size-6 !important;
+}
+
+// Non-heading sizes
+.font-size-xxl {
+	font-size: $font-size-0 !important;
+}
+
+.font-size-xl {
+	font-size: $font-size-1 !important;
+}
+
+.font-size-l {
+	font-size: $font-size-2 !important;
+}
+
+.font-size-m {
+	font-size: $font-size-3 !important;
+}
+
+.font-size-s {
+	font-size: $font-size-5 !important;
+}
+
+.font-size-xs {
+	font-size: $font-size-7 !important;
+}
+
+.font-size-xxs {
+	font-size: $font-size-9 !important;
+}
+
+// Weight
+.font-weight-light {
+	font-weight: $weight-light !important;
+}
+
+.font-weight-semilight {
+	font-weight: $weight-semilight !important;
+}
+
+.font-weight-normal {
+	font-weight: $weight-normal !important;
+}
+
+.font-weight-semibold {
+	font-weight: $weight-semibold !important;
+}
+
+.font-weight-bold {
+	font-weight: $weight-bold !important;
+}
+
+// Case
+.capitalized {
```

Comment:
text-transform: capitalize -> `text-transform-capitalize`, along with others

Reply:
on second thought I think we can remove these for now

Reply:
Don't we need them ? we use them a lot in docs-ui and i think we should include them . What do you think ?

Reply:
As discussed offline , we are not gonna use capitalize , lowercase and other classes .